examples/DatePicker.menu.jsx

```diff
@@ -3,32 +3,22 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 import * as React from 'react';
-import { DatePicker, IconButton, Popover } from '@itwin/itwinui-react';
-import { SvgCalendar } from '@itwin/itwinui-icons-react';
+import { DatePicker } from '@itwin/itwinui-react';
 
 export default () => {
   const [currentDate, setCurrentDate] = React.useState(new Date());
+  const [visible, setVisible] = React.useState(false);
 
   return (
     <>
-      <Popover
-        content={
-          <div className='demo-container'>
-            <DatePicker
-              showYearSelection
-              date={currentDate}
-              onChange={(date) => {
-                setCurrentDate(date);
-              }}
-            />
-          </div>
-        }
-        placement='bottom'
-      >
-        <IconButton label='Choose date'>
-          <SvgCalendar />
-        </IconButton>
-      </Popover>
+      <DatePicker
```

Comment:
This example should set `applyBackground={false}`.

Also I think it should be used within a `<Wrapper>` component (which you can create a the bottom of this file). This Wrapper component could be built using `Surface`, for applying background, border, padding, etc

Reply:
Will do.